src/azure.rs

```diff
@@ -261,6 +261,16 @@ impl PartialEq for AzColor {
     }
 }
 
+impl AzColor {
+    pub fn new(r: AzFloat, g: AzFloat, b: AzFloat, a: AzFloat) -> AzColor {
+         AzColor { r: r, g: g, b: b, a: a }
+    }
+
+    pub fn hack(color: AzColor) -> AzColor {
```

Comment:
Why is it necessary? It's unclear why you need to create a new AzColor where it's used.